src/main/java/org/jenkinsci/plugins/workflow/multibranch/template/ConfigDrivenWorkflowBranchProjectFactory.java

```diff
@@ -21,15 +21,20 @@
 public class ConfigDrivenWorkflowBranchProjectFactory extends AbstractWorkflowBranchProjectFactory {
     // TODO: Make this a parameter that users can adjust to their liking
     public static final String USER_DEFINITION_PATH = ".yourconfig.yml";
+    public static final String USER_DEFINITION_PIPELINE_PATH = "Jenkinsfile";
```

Comment:
Depending on whether we go with the name change for the field, it might be good to name this closer to what it is. 😄  

```suggestion
    public static final String DEFAULT_PIPELINE_TEMPLATE_PATH = "Jenkinsfile";
```